webapp/channels/src/components/admin_console/access_control/policies.scss

```diff
@@ -0,0 +1,163 @@
+.PolicyTable {
+    overflow: auto;
+
+    .DataGrid {
```

Comment:
Are you sure all of this customization is necessary? Given this component is used in so many other places in the system console, it seems weird that we'd need to override how it looks so heavily.

In general, it feels like there's a lot of CSS in this PR which are overriding styles on widely used classes like `.DataGrid`, and I can't tell when that's intentional or not